src/components/Checkbox.tsx

```diff
@@ -1,6 +1,31 @@
-import { Checkbox as MUICheckbox, FormControlLabel } from '@mui/material';
+import { Checkbox as MUICheckbox, FormControlLabel, Theme, useTheme } from '@mui/material';
 import React, { SyntheticEvent } from 'react';
 
+export const CheckboxStyle = (theme: Theme) => ({
+  backgroundColor: theme.palette.TwClrBaseWhite,
+  padding: theme.spacing(0, 1, 0, 0),
+  '& .MuiSvgIcon-root': {
+    fill: theme.palette.TwClrBrdrSecondary,
+  },
+  '&:hover > .MuiSvgIcon-root': {
+    fill: theme.palette.TwClrBrdrHover,
+  },
+  '&.Mui-checked > .MuiSvgIcon-root': {
+    fill: theme.palette.TwClrBgSelected,
+    color: theme.palette.TwClrBgSelected,
+  },
+  '&.MuiCheckbox-indeterminate > .MuiSvgIcon-root': {
+    fill: theme.palette.TwClrBgSelected,
+    color: theme.palette.TwClrBgSelected,
+  },
+  '&:focus > .MuiSvgIcon-root': {
```

Comment:
It looks like focus has a border radius in figma

Reply:
I didn’t add any border radius.. because the inner icon doesn’t have one. Added a comment in figma

Reply:
I added 2 more changes, removed some left margin and disabled animation on table checkboxes.